.gitignore

```
@@ -26,3 +26,5 @@
 /nbproject/
 /nbactions.xml
 /nb-configuration.xml
+.vscode/
+.factorypath
```

Do these exist only at the root of the project, or do they exist throughout the project?
If only at the root, they should be:

```suggestion
/.vscode/
/.factorypath
```

(I assume `.factorypath` is a regular file; if it's a directory, it should end in a slash also.)

---

No problem, '.vscode' directory only exists in the directory you open in the IDE, so it could technically be in anywhere, but usually only in the root. The '.factorypath' file can exist within any sub directory.

---

Okay, the way you have it should be fine, then.